pkg/apis/machineremediation/v1alpha1/replicas.go

```diff
@@ -0,0 +1,35 @@
+// NOTE: Boilerplate only.  Ignore this file.
```

Comment:
it has some unneeded comments
I also want to move this method to `utils` or under the `operator`, because it does not really relate to the API package, does it ok for you?